src/hooks/useContributions.ts

```diff
@@ -38,7 +38,98 @@ const mockData: UserContributions = {
   ],
 };
 
-export function useContributionUsers(): UserContributions {
+export function useContributionUsers(timeInput: {
+  start_date: string;
+  end_date: string;
+}): UserContributions {
+  const integrations = useCurrentCircleIntegrations();
+  const responses = useQueries(
+    integrations.data
+      ? integrations.data
+          .filter(
+            integration =>
```

Comment:
this integration filtering should be a func perhaps? or a lookup map of integrations to funcs to handle them , somehow there is a cleaner pattern here without these inline string compares

Reply:
yeah i've create consts that live in the integration creation file.